### ISSUE TITLE
Blocks Editor is not accessible to screen reader users

### ISSUE THREAD

**Issue by GoogleCodeExporter** [niteshmourya/app-inventor-for-android#2090]
```
The Blocks Editor is not accessible and therefore screen reader users cannot 
read the text presented by the editor or interact with the editor.

Using Mozilla Aurora and the NVDA screen reader.

Version: 101 Id: 5ca02efdf7d9

```

Original issue reported on code.google.com by `4RJa...@gmail.com` on 25 Apr 2012 at 4:28
